mempool/v0/clist_mempool.go

```diff
@@ -183,6 +183,16 @@ func (mem *CListMempool) TxsFront() *clist.CElement {
 	return mem.txs.Front()
 }
 
+// GetTxByKey is not supported by the v0 mempool but it is required to satisfy the mempool interface.
+func (mem *CListMempool) GetTxByKey(key types.TxKey) (types.Tx, bool) {
+	return nil, false
+}
+
+// GetTxEvicted is not supported by the v0 mempool but it is required to satisfy the mempool interface.
```

Comment:
[question] why is it difficult to add support for `GetTxEvicted` to the v0 mempool?

Reply:
sorry for repeating myself but why? I don't have enough info to understand why we're deprecating it instead of adding this feature and maintaining it. It hasn't been a maintenance burden before this feature so I'm wondering why adding support for `GetTxEvicted` is difficult for the v0 mempool.

Reply:
Sorry for the confusion. The idea of deprecating v0 mempool came from having to make changes to v0 every time we decided to alter the mempool interface. I don't think that it's difficult to maintain rather why maintain it if there's a possibility to remove something we're not currently supporting.

Reply:
> The idea of deprecating v0 mempool came from having to make changes to v0 every time we decided to alter the mempool interface.

Seems like we can do this PR without altering the mempool interface as suggested in a different comment 

> if there's a possibility to remove something we're not currently supporting.

We currently support the v0 mempool. In fact, it's the default on many other Cosmos chains because the v1 (prioritized) mempool was deprecated.